packages/components/src/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,163 @@
+.wrapper {
+  --avatar-sm: 1.25rem;
+  --avatar-md: 3rem;
+  --avatar-lg: 4.5rem;
+  --avatar-xl: 6rem;
+  --avatar-xxl: 7.75rem;
```

Comment:
Looking at this and noticed we actually slim down the number of variables we have here to a single one, something like `--avatar-x-y`.

We can then pass it into the width and height properties in the base class, then re-assign the value in all the modifiers like `.small`.